resources/security/mg.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
shall we move resources/security folder inside the adapter for now?

Reply:
Moved to adapter